src/olympia/amo/monitors.py

```diff
@@ -31,6 +31,28 @@ def execute_checks(checks: list[str]):
     return status_summary
 
 
+def web():
+    status = ''
+    response = requests.get('http://127.0.0.1:8002/__version__')
+
+    if response.status_code != 200:
+        status = f'Failed to ping web with status code: {response.status_code}'
+        monitor_log.critical(status)
+    return status, None
+
+
+def worker():
+    """
+    Check if the celery worker is healthy.
+    """
+    status = ''
+    result = celery.result.AsyncResult('healthcheck')
+    if result.failed():
```

Comment:
As discussed on slack: I'm Not sure this does anything useful, does it actually call the worker ?

Reply:
Fixed. `celery.current_app.inspect().ping()` faithfully returns None if the worker is shut down.